src/dataExplorer/components/OpenScript.tsx

```diff
@@ -42,7 +41,7 @@ const OpenScript: FC<Props> = ({onCancel, onClose}) => {
   const history = useHistory()
   const org = useSelector(getOrg)
 
-  const handleGetScripts = useCallback(async (name: string = '') => {
```

Comment:
Not a big deal - should the `name` in line 48 also be removed since it is not declared anywhere? When I test it on my local, the API call works fine